main.tf

```diff
@@ -173,7 +173,7 @@ resource "aws_cloudfront_distribution" "default" {
 
 module "dns" {
   source           = "git::https://github.com/cloudposse/terraform-aws-route53-alias.git?ref=tags/0.2.5"
-  enabled          = "${var.enabled}"
+  enabled          = "${var.enabled ? var.dns_enabled : false}"
```

Comment:
`enabled          = "${var.enabled == "true" && var.dns_enabled == "true" ? "true" : "false" }"`

Reply:
@joshmyers 
Thank you for review. I fixed it.